src/legacy/core_plugins/metrics/public/kbn_vis_type.js

```diff
@@ -23,11 +23,7 @@ import { ReactEditorControllerProvider } from './editor_controller';
 import { VisFactoryProvider } from 'ui/vis/vis_factory';
 import { defaultFeedbackMessage } from 'ui/vis/default_feedback_message';
 
-import { PANEL_TYPES } from '../../common/panel_types';
-
-// register the provider with the visTypes registry so that other know it exists
-import { VisTypesRegistryProvider } from 'ui/registry/vis_types';
-VisTypesRegistryProvider.register(MetricsVisProvider);
+import { PANEL_TYPES } from '../common/panel_types';
 
 export function MetricsVisProvider(Private) {
```

Comment:
and then you don't need this to be wrapped in Provider (as you no longer depend on Private)

Reply:
Unwrapped 👍